kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/lemmas.k

```diff
@@ -6,7 +6,7 @@ requires "evm-int-simplification.k"
 requires "int-simplification.k"
 requires "slot-updates.k"
 
-module LEMMAS [symbolic]
+module LEMMAS
```

Comment:
Removing `[symbolic]` so that the LLVM backend gets access to the helper functions from the slot updates lemmas.